src/cocoa/toga_cocoa/widgets/splitcontainer.py

```diff
@@ -14,48 +12,46 @@ def splitView_resizeSubviewsWithOldSize_(self, view, size: NSSize) -> None:
     def splitViewDidResizeSubviews_(self, notification) -> None:
         # If the window is actually visible, and the split has moved,
         # a resize of all the content panels is required.
-        if self._interface.window and self._interface.window._impl.isVisible:
-            # print("SPLIT CONTAINER LAYOUT CHILDREN", self._interface._containers[0]._impl.frame.size.width, self._interface._containers[1]._impl.frame.size.width)
-            self._interface._update_child_layout()
-            self._interface.on_resize()
+        if self.interface.interface.window and self.interface.interface.window._impl.native.isVisible:
```

Comment:
There's some naming inconsistency here - `self.interface` is actually the *implementation*.